src/main/java/com/leverx/employeestat/rest/controller/WorkController.java

```diff
@@ -0,0 +1,57 @@
+package com.leverx.employeestat.rest.controller;
+
+import com.leverx.employeestat.rest.dto.WorkDTO;
+import com.leverx.employeestat.rest.exception.NotValidUUIDException;
+import com.leverx.employeestat.rest.service.WorkService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.UUID;
+
+@RestController
+@RequestMapping("/api/works")
+public class WorkController {
```

Comment:
Maybe `Work` isn't good naming for it, maybe something like `UserProjectPosition`. Leaving it up to you.